examples/get-started/pure-js-mapbox/app.js

```diff
@@ -0,0 +1,56 @@
+import {Deck} from '@deck.gl/core';
```

Comment:
Nit: Why is this file new? Didn't we already have this example? Or git just failed to track the change?

Reply:
We renamed `pure-js` to `pure-js-mapbox`; and `pure-js-without-map` to `pure-js`.